vm/dependencies.go

```diff
@@ -33,7 +29,6 @@ type Config struct {
 	TransactionExecutionCores        int             `json:"transactionExecutionCores"`
 	StateFetchConcurrency            int             `json:"stateFetchConcurrency"`
 	MempoolSponsorSize               int             `json:"mempoolSponsorSize"`
-	StreamingBacklogSize             int             `json:"streamingBacklogSize"`
```

Comment:
If we're removing this here, we should remove it from the run script / e2e test fixture